.vscode/settings.json

```diff
@@ -1,7 +1,7 @@
 {
   "tailwindCSS.experimental.configFile": {
     "apps/storybook-react/tailwind.config.js": "packages/design-system-react/src/**",
-    "packages/design-system-twrnc-preset/tailwind.config.js": "packages/design-system-react-native/src/**"
+    "apps/storybook-react-native/tailwind-intellisense.config.js": "packages/design-system-react-native/src/**"
```

Comment:
Because of how twrnc works we don't use the tailwind.config.js for design tokens. Here we point to the same tailwind config that design system react does but inside of storybook-react-native

https://github.com/user-attachments/assets/13eea678-1e9f-45bf-a103-fe8c61d7c70e